js/utils/currency.js

```diff
@@ -136,12 +179,14 @@ export function formatPrice(price, currency) {
   }
 
   let convertedPrice;
+  // todo: this needs to take into account crypto listing curs which using the
+  // method below, would have most of them being considered as fiat.
```

Comment:
no, it should be 'being'.

Reply:
I'm not quite sure where the confusion is, but perhaps rephrasing to this would be clearer?

// todo: this needs to take into account crypto listing currency codes, which using the
// method below, would result in most of them considered as fiat.